tests/regression/rhbz-1708991/task.yaml

```diff
@@ -0,0 +1,32 @@
+summary: Check that snapd SELinux policy does not break systemd services with private mount ns
+
+systems: [fedora-*, centos-*]
+
+prepare: |
+    #shellcheck source=tests/lib/pkgdb.sh
+    . "$TESTSLIB"/pkgdb.sh
+
+    if [[ "$SPREAD_SYSTEM" == centos-7-* ]]; then
+        distro_install_package systemd-resolved
+    fi
+
+    getenforce > enforcing.mode
+
+    # Enable enforcing mode, our policy is already marked as permissive, so we
+    # will get audit entries but the program will not be stopped by SELinux
+    setenforce 1
+    ausearch --checkpoint stamp -m AVC || true
+
```

Comment:
Would it make sense to have a debug: section and print some extra selinux-related diagnostics in case we have any failures in the future? (FWTW I looked at existing selinux spread tests and we don't do that atm.)

Reply:
We already show the audit log in spread.yaml.